apps/src/weblab/WebLab.js

```diff
@@ -214,6 +214,11 @@ WebLab.prototype.init = function (config) {
 
   function onStartFullScreenPreview() {
     this.brambleHost.enableFullscreenPreview(() => {
+      // We always want to disable the inspector as we enter fullscreen preview,
+      // as it interferes with the preview display...
+      if (getStore().getState().inspectorOn) {
+        this.brambleHost.disableInspector();
```

Comment:
This is a bit confusing to me. Won't the redux state not reflect the actual state we want in the application?

Reply:
If redux says "inspector is on", then we disable it via `disableInspector()`. When/if that state changes (asynchronously), `onInspectorChanged()` will be called in `WebLab.js`, which updates redux to match the bramble state.

So, this is just like what happens in `WebLab.js` inside `onToggleInspector()`, except we only will disable the inspector mode in this case.